lib/solid_metrics.dart

```diff
@@ -2,14 +2,31 @@ library solid_metrics;
 
 import 'package:custom_lint_builder/custom_lint_builder.dart';
 import 'package:solid_metrics/metrics/cyclomatic_complexity_metric.dart';
+import 'package:solid_metrics/models/cyclomatic_complexity_parameters.dart';
 
 /// creates plugin
 PluginBase createPlugin() => _SolidMetrics();
 
 /// Solid metric linter
 class _SolidMetrics extends PluginBase {
+  /// Get params of lint rule
+  Map<String, Object?>? _getParamsForLintRule(
+    CustomLintConfigs configs,
+    String code,
+  ) {
+    return configs.rules[code]?.json;
+  }
+
   @override
   List<LintRule> getLintRules(CustomLintConfigs configs) => [
-        const CyclomaticComplexityMetric(),
+        CyclomaticComplexityMetric(
+          additionalParameters: CyclomaticComplexityParameters.fromJson(
+            _getParamsForLintRule(
+                  configs,
+                  CyclomaticComplexityMetric.lintCode.name,
+                ) ??
+                {},
```

Comment:
I think we should not include it in the list if the user does not add it to the list of enabled rules.